starter-code/views/auth/signup.ejs

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="stylesheets/style.css">
+  <title>Sign Up</title>
+</head>
+
+<body>
+  <form id="form" action="/signup" method="POST">
+    <label for="username">Username</label><br>
+    <input type="text" name="username" 
+      placeholder="AnakinSkywalker" required>
+
+    <br><br>
```

Comment:
Put more of these and it makes the sound of a chainsaw 😛 .

Joking aside, there are several things that you could do better here:

- Spacing is better handled by CSS instead of adding linebreaks
- The `for` attribute of the `label` tags looks for tags with the same `id`, not the same `name`; Instead I encourage you to put the `input` *inside* the label so you don't have to use `for` (yes, we can do that 🎉 ).
- Are you sure you want a clear-text placeholder for the password?
- You can give a more meaningful `id` for your form.

Here is a suggestion below:

```
<form id="signup-form" action="/signup" method="POST">
    <label>Username: <input type="text" name="username" placeholder="AnakinSkywalker" required></label>
    <label>Password: <input type="password" name="password" placeholder:"*******"></label>
</form>
```

And in your CSS:

```
#signup-form label {
    display: block; /* Because it is now a block you'll automatically have a line break between each field */
    margin-bottom: 0.8rem; /* If you want a little bit of whitespace between those */
}
#signup-form label input {
     display: block; /* If you want a line break between the input field and the actual label (although I advise against it) */
}
```